api/internal/target/kusttarget.go

```diff
@@ -366,6 +366,9 @@ func (kt *KustTarget) accumulateResources(
 	for _, path := range paths {
 		// try loading resource as file then as base (directory or git repository)
 		if errF := kt.accumulateFile(ra, path, origin); errF != nil {
+			if strings.HasPrefix(path, "https://") || strings.HasPrefix(path, "http://") {
```

Comment:
I'm trying to understand why you have this check. For example, I can have the following kustomization with a remote base, which currently works fine:

```
resources:
   - https://github.com/config-sync-examples/helm-components.git
```

but with this check, it seems to me that such a case will no longer work. Is my understanding correct?

Reply:
You're right, I'll fix that.

Reply:
Fixed!